cmd/corednsmonitor/corednsmonitor.go

```diff
@@ -80,6 +81,7 @@ func main() {
 	rootCmd.Flags().IPSlice("cloud-ext-lb-ips", nil, "IP Addresses of Cloud External Load Balancers for OpenShift API")
 	rootCmd.Flags().IPSlice("cloud-int-lb-ips", nil, "IP Addresses of Cloud Internal Load Balancers for OpenShift Internal API")
 	rootCmd.Flags().IPSlice("cloud-ingress-lb-ips", nil, "IP Addresses of Cloud Ingress Load Balancers")
+	rootCmd.Flags().String("platform-type", "", "PlatformType of cloud")
```

Comment:
doc nit: this field is not about a cloud, it's just a platform type so more appropriate description is something like `platformType provides the platform name`